### ISSUE TITLE
[BUG] `DialogContent` requires a `DialogTitle`

### ISSUE THREAD

**Issue by filiabel** [formbricks/formbricks#3098]
### Issue Summary

Navigating to any dialog/modal throws error and warning in the browser console.

>**[ERROR]** `DialogContent` requires a `DialogTitle` for the component to be accessible for screen reader users.
>
>If you want to hide the `DialogTitle`, you can wrap it with our VisuallyHidden component.
>
>For more information, see https://radix-ui.com/primitives/docs/components/dialog

>**[WARNING]** Missing `Description` or `aria-describedby={undefined}` for {DialogContent}.

### Steps to Reproduce

1. Open browser
2. Navigate to a page with modal. E.g. `Surveys > Some Survey > Share Button`
3. Observe console

### Expected behavior

No error logs

### Screenshots

![image](https://github.com/user-attachments/assets/ba7eb9a1-366a-4d20-b170-204326c1a95e)

### Environment

- [ ] Formbricks Cloud (app.formbricks.com)
- [X] Self-hosted Formbricks

### Desktop (please complete the following information)

```markdown
- OS: Win 10
- Formbricks v2.5.0
- Browser: Chrome
```

**Comment by jobenjada** [comment]
thanks for opening the issue, feel free to self-assign in case your interested in solving it 🤓  `/assign`

**Comment by jobenjada** [comment]
mh something seems broken with oss.gg, let me have a look :)

**Comment by jobenjada** [comment]
@RajuGangitla I've given you the other issue you asked for and assigned @SaiSawant1 to this one :)

**Comment by SaiSawant1** [comment]
@jobenjada completely missed this. I was busy with school. My bad will fix this soon.

**Comment by jobenjada** [comment]
Thanks to everyone involved in fixing this 👋
